src/main/java/menion/android/whereyougo/preferences/PreferenceValues.java

```diff
@@ -193,8 +193,6 @@ public static Activity getCurrentActivity() {
     }
 
     public static void setCurrentActivity(Activity activity) {
-        if (PreferenceValues.currentActivity == null && activity != null)
-            MainApplication.appRestored();
```

Comment:
This is result of the change in MainApplication, therefore it should included in their PR.